src/api/rest/server/common/utility.go

```diff
@@ -1,14 +1,55 @@
 package common
 
 import (
+	"errors"
 	"fmt"
 	"net/http"
 
+	"github.com/astaxie/beego/validation"
 	"github.com/labstack/echo/v4"
 
 	"github.com/cloud-barista/cb-tumblebug/src/core/common"
 )
 
+type Existence struct {
+	Exists bool `json:"exists"`
+}
+
+func SendExistence(c echo.Context, httpCode int, existence bool) error {
+	return c.JSON(httpCode, Existence{Exists: existence})
+}
+
+type Status struct {
+	Message string `json:"message"`
+}
+
+func SendMessage(c echo.Context, httpCode int, msg string) error {
+	return c.JSON(httpCode, Status{Message: msg})
+}
+
+func Send(c echo.Context, httpCode int, json interface{}) error {
+	return c.JSON(httpCode, json)
+}
+
+func Validate(c echo.Context, params []string) error {
+	valid := validation.Validation{}
+
+	for _, name := range params {
+		valid.Required(c.Param(name), name)
+	}
+
+	if valid.HasErrors() {
+		for _, err := range valid.Errors {
+			return errors.New(fmt.Sprintf("[%s]%s", err.Key, err.Error()))
+		}
+	}
+	return nil
+}
+
+func Healthy(c echo.Context) error {
+	return c.String(http.StatusOK, "Hello, World! This is Cloud-Barista CB-Tumblebug")
+}
```

Comment:
healthy -> healthy 반영시

```suggestion
func Health(c echo.Context) error {
	return c.String(http.StatusOK, "The API server of CB-Tumblebug is alive")
}
```

Reply:
func Healthy 명칭도
func GetHealth 와 같은 형태가 더 낮지 않을까 싶네요.. ^^

Reply:
의견 감사합니다. 반영했습니다!